crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -495,6 +467,65 @@ impl Space {
             }
         });
 
+        if let NetworkType::QuicBootstrap = &config.network_type {
+            // spawn the periodic bootstrap pull
+            let i_s_c = i_s.clone();
+            let evt_s_c = evt_sender.clone();
+            let bootstrap_service = config.bootstrap_service.clone();
+            let space_c = space.clone();
+            tokio::task::spawn(async move {
+                const START_DELAY: std::time::Duration = std::time::Duration::from_secs(1);
+                const MAX_DELAY: std::time::Duration = std::time::Duration::from_secs(60 * 60);
+
+                let mut delay_len = START_DELAY;
+
+                loop {
+                    tokio::time::sleep(delay_len).await;
```

Comment:
So, this loop never ends, and exponentially backs off to one minute, at which point it pings the bootstrap server every minute forever?

I know this isn't part of the change in this PR. Seems fine for now but maybe we need to change this once we expect more Holochain users? If we have 100k conductors, that's 1600 requests per second total to the bootstrap server.

Reply:
It backs off to 1 hour, then pings it once per hour. I think once per hour is reasonable.

Reply:
Oops, yeah. Rushed that last message.